collector/src/execute.rs

```diff
@@ -561,9 +571,17 @@ impl<'a> MeasureProcessor<'a> {
         cid: database::ArtifactIdNumber,
         self_profile: bool,
     ) -> Self {
-        // Check we have `perf` available.
-        let has_perf = Command::new("perf").output().is_ok();
-        assert!(has_perf);
+        // Check we have `perf` or (`xperf.exe` and `tracelog.exe`)  available.
+        if cfg!(unix) {
+            let has_perf = Command::new("perf").output().is_ok();
+            assert!(has_perf);
+        } else {
+            let has_xperf = Command::new(env::var("XPERF").unwrap_or("xpref.exe".to_string())).output().is_ok();
```

Comment:
XPERF vs. xpref (swapping pref and perf) seems a bit odd here -- by mistake?

Reply:
Good catch! Fixed.